extensions/adobe/experience/adcloud/search/account-all.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "meta:license": [
+        "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://ns.adobe.com/experience/adcloud/search/account-all",
+    "title": "Adobe AdCloud Search Account Entity Full Extension",
+    "type": "object",
+    "meta:extensible": true,
+    "meta:abstract": true,
+    "meta:intendedToExtend": [
+        "https://ns.adobe.com/xdm/context/profile-adcloud"
+    ],
+    "description": "Adobe AdCloud Search Account Entity Full Extension. Contains all Solution added fields.",
+    "definitions": {
+        "account-all": {
+            "properties": {
+                "xdm:account": {
```

Comment:
Because this mixin (has the intendedToExtend) exists inside the adcloud namespace, all top level fields must be under an adcloud namespace rather than xdm. 
This can be solved two different ways:
Option 1: Instead of xdm:account the field should be https://ns.adobe.com/experience/adcloud/account
Option 2: You create a top level field named https://ns.adobe.com/experience/adcloud/account and then nest all the xdm: fields under it. 

See an example here - https://github.com/adobe/xdm/blob/master/extensions/adobe/experience/campaign/experienceevent-profile-push-details.schema.json#L19
2. You create a top level field named https://ns.adobe.com/experience/adcloud/account

Reply:
Thanks @cmathis. We chose option 1 and modified all top level fields under an ad-cloud namespace